packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2-base.ts

```diff
@@ -1,7 +1,73 @@
 import { Resource, Annotations } from 'aws-cdk-lib';
 import { IVpc, ISubnet, SubnetSelection, SelectedSubnets, EnableVpnGatewayOptions, VpnGateway, VpnConnectionType, CfnVPCGatewayAttachment, CfnVPNGatewayRoutePropagation, VpnConnectionOptions, VpnConnection, ClientVpnEndpointOptions, ClientVpnEndpoint, InterfaceVpcEndpointOptions, InterfaceVpcEndpoint, GatewayVpcEndpointOptions, GatewayVpcEndpoint, FlowLogOptions, FlowLog, FlowLogResourceType, SubnetType, SubnetFilter, CfnVPCCidrBlock } from 'aws-cdk-lib/aws-ec2';
 import { allRouteTableIds, flatten, subnetGroupNameFromConstructId } from './util';
-import { IDependable, Dependable, IConstruct } from 'constructs';
+import { IDependable, Dependable, IConstruct, DependencyGroup } from 'constructs';
+import { EgressOnlyInternetGateway, InternetGateway, NatConnectivityType, NatGateway, NatGatewayOptions, Route, VPNGatewayV2 } from './route';
+import { ISubnetV2 } from './subnet-v2';
+
+/**
+ * Options to define EgressOnlyInternetGateway for VPC
+ */
+export interface EgressOnlyInternetGatewayOptions{
+  /**
+   * List of subnets where route to EGW will be added
+   * @default no route created
+   */
+  readonly subnets?: SubnetSelection[];
+
+  /**
+   * Destination Ipv6 address for EGW route
+   * @default '::/0' all Ipv6 traffic
```

Comment:
```suggestion
   * Destination Ipv6 address for EGW route
   *
   * @default - '::/0' all Ipv6 traffic
```
Can you clarify what this default value means? It feels a bit confusing when I read it.

Reply:
default means setting up outbound to all IPv6 address for which the range is '::/0'